test/verify/check-services

```diff
@@ -836,6 +836,68 @@ WantedBy=default.target
         b.click(".modal-dialog .btn-danger")
         self.check_service_details(["Masked"], ["Allow running (unmask)"], False, onoff=False)
 
+    @skipImage("Implemented in #12585", "rhel-8-0-distropkg", "rhel-8-1-distropkg")
```

Comment:
Likewise, rhel-8-0-distropkg can go now. This can happen in a follow-up too, if this is otherwise ready.

Reply:
Removed, yay! :-)